catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -327,4 +328,8 @@
     <service ref="workspaceQueryService"
              interface="org.codice.ddf.catalog.ui.query.monitor.api.WorkspaceQueryService"/>
 
+    <bean id="featureService" class="org.codice.ddf.catalog.ui.query.geofeature.StubFeatureService"></bean>
```

Comment:
nit : ` <bean id="featureService" class="org.codice.ddf.catalog.ui.query.geofeature.StubFeatureService"/>`.  Can remove the closing bean element